ocrd/ocrd/decorators/__init__.py

```diff
@@ -36,9 +35,10 @@ def ocrd_cli_wrap_processor(
     list_resources=False,
     **kwargs
 ):
-    if not sys.argv[1:]:
+    if not (mets or working_dir or 
+            dump_json or dump_module_dir or help or version or show_resource or list_resources):
         processorClass(workspace=None, show_help=True)
-        sys.exit(1)
```

Comment:
Why remove the sys.exit? They can still be caught (`except SystemExit`) in tests but with a return value, processors will need to adapt their CLI wrapping code.

Reply:
Yes, I was probably mistaken about that. CliRunner surely can handle (i.e. catches) SystemExit, and merely returning may not be the right thing in all contexts.

Reply:
I have reverted that bit.